templates/cron_staleness_check.erb

```diff
@@ -10,7 +10,9 @@ function get_context {
   # Scans syslog for a given logger tag where we might have additional
   # information about why a cron job failed.
   echo "Some potentially helpful context from /var/log/messages: (may be empty)"
-  grep "${1}: " /var/log/messages | tail | head -c 1024
+  echo "{code}
```

Comment:
Aug, you got me. Maybe I should test :(

But in theory would you give a shipit to such a change?